specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/config/client.event.yaml

```
@@ -24,7 +24,8 @@ telemetry:
         events:
           - qname: test.app0
             id: binding.kafka.api.version.rejected
-            message: API_VERSION_REJECTED 32 0
+            name: BINDING_KAFKA_API_VERSION_REJECTED
+            message: 32 0
```

Do you think we should map these API keys to the human readable equivalents for better readability?
See https://kafka.apache.org/protocol.html#protocol_api_keys

```suggestion
            message: DESCRIBE_CONFIGS (32) version 0
```

---

Yes definitely. Is there an enum or library with them already or do we need to map them manually?

---

I just talked with @attilakreiner about the general design of these message outputs and we might want to address the human readability for all of them. Particulary since we are planning to allow a `json` format option. This will help with the `MODEL_AVRO_VALIDATION_FAILED` and `BINDING_TLS_TLS_FAILED` events. 

- https://github.com/aklivity/zilla/issues/1012